Mathlib/Dynamics/PeriodicPts/Lemmas.lean

```diff
@@ -32,6 +32,13 @@ theorem minimalPeriod_eq_prime {p : ℕ} [hp : Fact p.Prime] (hper : IsPeriodicP
   (hp.out.eq_one_or_self_of_dvd _ hper.minimalPeriod_dvd).resolve_left
     (mt minimalPeriod_eq_one_iff_isFixedPt.1 hfix)
 
+theorem minimalPeriod_eq_prime_iff {p : ℕ} [hp : Fact p.Prime] :
```

Comment:
Please put this result before `minimalPeriod_eq_prime` and deduce the proof of `minimalPeriod_eq_prime` from it. You can also add a comment like: `The forward direction of minimalPeriod_eq_prime_iff` (well, I guess it's more the `backward direction` ;)

Same thing for `orderOf_eq_prime` and `orderOf_eq_prime_iff`

Reply:
Sorry, I forgot to submit my review. The new comments should be visible now.